### PR TITLE
Javascript sequelize raw query

### PR DESCRIPTION
**Pull request by 0xdbe**
Add rule to prevent query using concatenation with Sequelize.

### CODE REVIEW

**Review by inkz — APPROVED** [review]
lgtm

File: javascript/sequelize/security/audit/sequelize-raw-query.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+rules:
+  - id: sequelize-raw-query
+    message: >
+       Avoiding SQL string concatenation: untrusted input concatinated with raw
+       SQL query can result in SQL Injection. Data remplacement or data binding
```
Comment (ievans) [inline javascript/sequelize/security/audit/sequelize-raw-query.yaml:6]:
```suggestion
       SQL query can result in SQL Injection. Data replacement or data binding
```
great message, just fixing a typo.

**Comment by ievans** [comment]
@minusworld do we have a consistent guide yet on line/word wrapping?

@aryx this seems like an ideal case for `taint` mode

**Comment by aryx** [comment]
cc @IagoAbal who is working a bit on the tainting mode. Might be a good test case.

File: javascript/sequelize/security/audit/sequelize-raw-query.yaml (hunk 1 above)
Comment (inkz) [reply javascript/sequelize/security/audit/sequelize-raw-query.yaml:6]: my bad https://github.com/returntocorp/semgrep-rules/pull/1180

**Pull request merged**